docusaurus/src/components/HeaderDecoration.module.css

```
@@ -5,46 +5,49 @@
   margin-right: 10px;
 }
 
+div.connectorMetadataCallout  {
+  margin-bottom: 15px;
+}
+
 .connectorMetadata {
-  background: var(--ifm-color-info-contrast-background);
-  border-radius: 2px;
   width: 100%;
-  font-size: smaller;
   margin: 0;
-  padding: 5px;
-  margin-bottom: 15px;
+  display: flex;
+  flex-direction: column;
+  row-gap: 2px;
 }
 
-.connectorMetadata > div {
+.metadataStat {
   display: flex;
   align-items: center;
-  gap: 0.2em;
+  font-size: 0.75rem;
+  font-weight: var(--ifm-font-weight-normal);
 }
 
-.connectorMetadata dt {
-  display: inline;
-  font-weight: bold;
-}
+.metadataStatLabel {
+  display: inline-flex;
+  align-items: center;
 
-.connectorMetadata dd {
-  display: inline;
-  margin: 0;
+  font-weight: var(--ifm-font-weight-semibold);
+  margin-right: 10px;
 }
 
-.connectorMetadata dt::after {
-  content: ": ";
+.metadataStatValue {
+  display: inline-flex;
+  align-items: center;
+  margin: 0;
```

```suggestion
  margin: 0;

  a {
    display: inline-flex;
    align-items: center;
  }
```

This should fix the vertical alignment issue I mentioned [here](https://github.com/airbytehq/airbyte/pull/38296/commits/03132919d6f25f1cc99d5235dda3b67c6f8bc6f6#r1604061039)

---

Oh shoot did that last change not fix it?

Looking again!